src/skins/o2-new.tsx

```diff
@@ -0,0 +1,356 @@
+import {O2_NEW_SKIN} from './constants';
+import {applyAlpha} from '../utils/color';
+
+import type {GetKnownSkin, KnownSkin} from './types';
+
+export const palette = {
+    beyondBlue: '#0050FF',
+    beyondBlue10: '#E5EDFF',
+    beyondBlue15: '#CCDCFF',
+    beyondBlue30: '#80A7FF',
+    beyondBlue40: '#4D84FF',
+    beyondBlue45: '#0A73EB',
+    beyondBlue55: '#0044D9',
+    beyondBlue70: '#0038B2',
+    beyondBlue90: '#001033',
+    darkBlue: '#00008C',
+    o2BlueLight: '#82DCFA',
+    o2BlueLight30: '#C0EEFD',
+    o2BlueLight35: '#B4EAFC',
+    o2Green: '#00DC7D',
+    o2Green10: '#E5FBF2',
+    o2Green40: '#4CE7A4',
+    o2Green80: '#006338',
+    o2Yellow: '#FADC00',
+    o2Orange: '#FFA55A',
+    o2Orange10: '#FFF6EE',
+    o2Orange40: '#FFC08B',
+    o2Orange75: '#996336',
+    o2Pink: '#FA96FF',
+    o2Pink15: '#FEEAFF',
+    o2Pink30: '#FDCAFF',
+    o2Pink80: '#704373',
+    o2Red: '#FF5A5A',
+    o2Red10: '#FFEEEE',
+    o2Red20: '#FFCDCD',
+    o2Red40: '#FF8B8B',
+    o2Red45: '#FF7B7B',
+    o2Red60: '#CC4848',
+    o2Red65: '#BF4343',
+    grey20: '#F3F3F5',
+    grey30: '#D9D9DD',
+    grey40: '#B4B4BE',
+    grey45: '#8C8C9A',
+    grey60: '#6E6E77',
+    grey80: '#3C3C46',
+    black: '#00001E',
+    white: '#FFFFFF',
+    darkModeBlack: '#121212',
+    darkModeGrey: '#1E1E1E',
+    darkModeGrey6: '#2B2C2E',
+    darkModeBeyondBlue: '#1A62FF',
+    darkModeBeyondBlueDark: '#0048E5',
+};
+
+export const getO2NewSkin: GetKnownSkin = () => {
+    const skin: KnownSkin = {
+        name: O2_NEW_SKIN,
+        colors: {
+            background: palette.white,
+            backgroundAlternative: palette.grey20,
+            backgroundBrand: `linear-gradient(180deg, ${palette.darkBlue} 0%, ${palette.beyondBlue} 64%, ${palette.beyondBlue45} 100%)`,
+            backgroundBrandSecondary: palette.beyondBlue,
+            backgroundContainer: palette.white,
+            backgroundContainerHover: applyAlpha(palette.darkModeBlack, 0.03),
+            backgroundContainerPressed: applyAlpha(palette.darkModeBlack, 0.05),
+            backgroundContainerBrand: `linear-gradient(180deg, ${applyAlpha(
```

Comment:
Please, do another pull from mistica-design to fix this token. I just fixed it and shouldn't have opacities